### PR TITLE
Add integration tests for views

### PR DESCRIPTION
**Pull request by jchernandez87**
### Changes: 😄 

- Add integration test for Sign up view.
- Add integration test for User index view.
- Add integration test for User show view.
- Add integration test for Post index view.
- Add integration test for Post show view.

### CODE REVIEW

File: spec/views/sing_in_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    click_button 'Log in'
+    expect(page).to have_content 'Invalid Email or password.'
+  end
+
+  it 'fails to sign with incorrect password' do
+    visit '/users/sign_in'
+    within('#new_user') do
+      fill_in 'user_email', with: 'another@user.com'
+      fill_in 'user_password', with: 'newpassword'
+    end
```
Comment (andersonlebon) [inline spec/views/sing_in_spec.rb:30]: image

- [ ]  This :point_up: is the statistic your text, as you can see now some test cases error. Kindly let's go ahead and fix this est error.
- [ ] Also Your application needs to run properly even though you add tests.. Now I am not redirected to the login form

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @jchernandez87 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jchernandez87** [comment]
tests

**Review by rukundoeric — APPROVED** [review]
Hi @jchernandez87,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**